cli/commands/delete_lease.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/codegangsta/cli"
+	"github.com/deis/k8s-claimer/htp"
+)
+
+// DeleteLease is a cli.Command action for deleting a lease
+func DeleteLease(c *cli.Context) {
+	server := c.GlobalString("server")
+	if server == "" {
+		log.Fatalf("Server missing")
+	}
+	if len(c.Args()) < 1 {
+		log.Fatalf("Lease token missing")
+	}
+	leaseToken := c.Args()[0]
```

Comment:
Would it be worth it to inspect Env to see if `TOKEN` is already set to a value (perhaps from a `lease create` command in combination from sourcing the returned `export` statements)?  If set, we could use this as the default, not requiring any value in `Args()[0]`